src/services/api.service.ts

```diff
@@ -721,8 +721,8 @@ export class ApiService {
   /*
    * Api Keys
    */
-  verifyApiKey(apiId: string, apiKey: string): IHttpPromise<any> {
-    return this.$http.post(`${this.Constants.env.baseURL}/apis/${apiId}/keys/_verify?apiKey=${apiKey}`, {});
+  verifyApiKey(apiId: string, applicationId: string, apiKey: string): IHttpPromise<any> {
+    return this.$http.get(`${this.Constants.env.baseURL}/keys/_canCreate?key=${apiKey}&application=${applicationId}&api=${apiId}`, {});
```

Comment:
maybe we can use this syntax ?
```
$http.get(path, {
    params: { user_id: user.id }
});
```

Reply:
done, it looks better like that, thanks :+1: